pkg/cluster/k8sres.go

```diff
@@ -1305,6 +1306,7 @@ func (c *Cluster) generateStatefulSet(spec *acidv1.PostgresSpec) (*appsv1.Statef
 
 func (c *Cluster) generatePodAnnotations(spec *acidv1.PostgresSpec) map[string]string {
 	annotations := make(map[string]string)
+	// TODO annotations[rollingUpdatePodAnnotationKey] = strconv.FormatBool(false)
```

Comment:
I am likely not a fan of this approach.

I suggest we have a good point, where we explicitly "patch" the pods.